src/modules/navigator/mission_block.cpp

```diff
@@ -307,6 +308,19 @@ MissionBlock::is_mission_item_reached()
 				mission_acceptance_radius = _navigator->get_acceptance_radius();
 			}
 
+			/* for vtol back transition calculate acceptance radius based on time and ground speed */
+			if (_mission_item.vtol_back_transition) {
+
+			    float groundspeed = sqrtf(_navigator->get_global_position()->vel_n * _navigator->get_global_position()->vel_n +
```

Comment:
The problem is that the back transition time is not right for VTOLs flying with airspeed. We've just started testing another option that you could just add here instead, using a scaling factor for the already ground speed dependent dynamic radius:
```
			/* scale acceptance radius if requested */
			if (_mission_item.vtol_acceptance_radius) {
				mission_acceptance_radius *= _param_vtol_bt_rad_scale.get();
			}
```